apps/squeaknode/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: yzernik/squeaknode:v0.1.21@sha256:8555988fd28b4d9e6533827510f8f618182d10de3f6e06b988d89048617615ba
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8774:8774
+      - 12994:12994
+    volumes:
+      - ${APP_DATA_DIR}/sqk:/sqk
```

Comment:
This directory should be committed into source control, even if it's empty.

Git doesn't actually allow empty directories so you can touch a file at `sqk/.gitkeep` and commit it.

When installing your app Umbrel will strip out the `.gitkeep` file.

The reason to do this is that if the directory doesn't exist Docker will create it the first time your container is run as root and only processes running as root will be able to read from it.

If you commit the dir you can set the correct permissions and they will be respected.

Reply:
✓ added the empty sqk directory with gitkeep